Dockerfile

```diff
@@ -1,7 +1,13 @@
-FROM jinaai/jina:master
+FROM jinaai/jina:master as base
 
-ADD *.py *.yml requirements.txt ./
+COPY . ./image_normalizer/
+WORKDIR ./image_normalizer
 
-RUN pip install -r requirements.txt
+RUN pip install .
 
-ENTRYPOINT ["jina", "pod", "--uses", "config.yml"]
\ No newline at end of file
+FROM base
+RUN pip install -r tests/requirements.txt
+RUN pytest tests
+
+FROM base
+ENTRYPOINT ["jina", "executor", "--uses", "config.yml"]
```

Comment:
Have you tried using the Executor via the docker image? I would imagine, this does not work. The reason: The `image_normalizer` files will not be copied, since you start from base and not from after `pip install`.

Reply:
Yes, this is working as it is. The FROM base in the docker image uses the previous commands as well.